starter-code/iteration-1/main.ts

```diff
@@ -1,11 +1,27 @@
 // Iteration 1
 // 1. Create an array of strings
+let TodoList: Array<string> = [];
 
 // 2. Create a function to add a task in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
+function addTask(task:string) {
+  TodoList.push(task)
+  console.log("Task added!");
+  return TodoList.length;
+}
 
 // 3. Create a function to list all tasks, it must show in the console de task.
+function listAllTasks() {
+  for (var i = 0; i <TodoList.length; i++){
```

Comment:
you could improve this with a forEach loop:
`this.tasks.forEach((task, index) => {
    console.log(task);`